lib/endpoint/find-parameters.js

```diff
@@ -342,21 +359,114 @@ function mapToInput (state) {
   })
 }
 
-function formatParam (rawParam) {
-  const param = {
-    name: rawParam.name,
-    schema: { type: rawParam.type },
-    required: rawParam.required,
-    description: rawParam.description || `${rawParam.name} parameter`
+function formatParam (param) {
+  const { method, path, operation } = this
```

Comment:
I try to avoid `this` to avoid hard to find bugs and make the code easier to comprehend. I'd instead pre-bind the format param function

```diff
- .map(formatParam, route)
+ .map(formatParam.bind(null, route))
```

Or pass it explicitly

```diff
- .map(formatParam, route)
+ .map(rawParam => formatParam(route, rawParam))
```

Probably the latter is the most readable?

Reply:
nevermind looks like you removed that in commits after :)